packages/documentation/src/content/docs/telemetry/integrating.md

```diff
@@ -9,6 +9,7 @@ In order to do so, the integrating ASE must deploy its own OpenTelemetry collect
 ## Rafiki Telemetry Environment Variables
 
 - `ENABLE_TELEMETRY`: boolean, defaults to `true`. Enables the telemetry service on Rafiki.
+- `LIVENET`: boolean. Should be set to `true` on production environments. If it is not set, it will default to `false`, and metrics will get sent to the testnet otel-collector
```

Comment:
```suggestion
- `LIVENET`: boolean. Should be set to `true` on production environments dealing with real money. If it is not set, it will default to `false`, and metrics will get sent to the testnet otel-collector
```